Makefile

```
@@ -285,7 +285,7 @@ undeploy: kustomize ## Undeploy controller from the K8s cluster specified in ~/.
 .PHONY: install-external-crds
 install-external-crds: ## Install external CRDs from milo repository.
 	@echo "Installing MachineAccount CRDs from milo repository..."
-	@kubectl apply -k https://github.com/datum-cloud/milo/config/crd/bases/iam?ref=212-add-the-ability-to-activate-and-deactivate-user-accounts-in-milo || { \
+	@kubectl apply -k https://github.com/datum-cloud/milo/config/crd/bases/iam?ref=integration/user-deactivation || { \
```

Let's point this at the stable version.

```suggestion
	@kubectl apply -k https://github.com/datum-cloud/milo/config/crd/bases/iam?ref=v0.2.0 || { \
```